exporter/elasticsearchexporter/logs_exporter.go

```diff
@@ -61,8 +61,7 @@ func newLogsExporter(logger *zap.Logger, cfg *Config) (*elasticsearchLogsExporte
 		maxAttempts = cfg.Retry.MaxRequests
 	}
 
-	// TODO: Apply encoding and field mapping settings.
-	model := &encodeModel{dedup: true, dedot: false}
+	model := &encodeModel{dedup: cfg.Mapping.Dedup, dedot: cfg.Mapping.Dedot}
```

Comment:
Could you add a few tests to cover this change?

Reply:
@evan-bradley I didn't add tests because the current test code doesn't have functionality to test internal fields, and I felt adding this to the test code was out of scope for such a minor change.

But, if you think that's still in scope for this PR I can add some changes.

Reply:
You're right, it doesn't look like there are any existing test cases that could be trivially adapted for this.

In my opinion unit tests usually pair well with bug fixes, though I agree with you that the scope of the code changes here is small. If you would be willing, I think a few tests that call `push[Trace|Logs]Data` and verify the resulting JSON object in the `newESTestServer` handler function would still be good additions here.

Reply:
No problem, thanks for adding the tests.